src/Auth/Contracts/Authenticable.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Auth\Contracts;
+
+interface Authenticable
```

Comment:
We need to find a better name

Reply:
I think `HasIdentity` was my preference, but I'm open for other suggestions

Reply:
I'll update to `HasIdentity` for now then

Reply:
@brendt - My only complaint with `HasIdentity` is that it seems to conflict with our trait naming convention.

Reply:
AuthenticatableInterface ?

Reply:
@aidan-casey good point. Did you suggest `Identifiable`? That's fine for me as well. I don't love it, but I don't hate it

Reply:
I'll be using `Identifiable` for the interface and I'm going to create a trait for Models.
Should I name it `HasIdentity` then? Or `IsIdentifiable`?
Should I put this trait in the `Database` folder or leave it in the `Auth` one @brendt?

Another thing I'd like to suggest is to start using something like I'm doing to structure the code in the folders, moving interfaces into a `Contracts` folder (or another name), doing the same for `Traits`, I'm thinking on creating them into a `Concerns` folder (or another name).

What do you think, @brendt?

Reply:
`HasIdentity` and leave it in the `Auth` folder 👍

I was thinking about the same topic when it comes to structuring, but approach it from a different angle. My intuition says that all classes/interfaces/attributes/enums/… that are accessed by external users should live in the root namespace. 

I don't know if that'll work in the long run though. For now, I'd keep things together in places where it makes sense, and review this topic again at a later point

Reply:
Nice, I'll use this name then.

Ok, I'll follow this approach then, the classes that are going be used/referenced/imported in the user code I'll move them to the root namespace for now.

Thanks for the help on this, @brendt! 💪